### PR TITLE
Role no longer subs thing in meta ontology

### PR DESCRIPTION
**Pull request by fppt**

### CODE REVIEW

File: grakn-test/src/test/java/ai/grakn/property/TypePropertyTest.java [hunk 1]
```diff
@@ -189,6 +190,7 @@ public void whenSettingATypeAbstractFlag_TheTypesAbstractFlagIsSet(@Meta(false)
     @Property
     public void whenAnOntologyElementHasADirectSuper_ItIsADirectSubOfThatSuper(
             @Open GraknGraph graph, @FromGraph OntologyConcept ontologyConcept) {
+        assumeFalse(Schema.MetaSchema.ROLE.getLabel().equals(ontologyConcept.getLabel()));
```
Comment (aelred) [inline grakn-test/src/test/java/ai/grakn/property/TypePropertyTest.java]: Does the test fail without this assumption?

Comment (fppt) [inline grakn-test/src/test/java/ai/grakn/property/TypePropertyTest.java]: yep. `role` no longer has a super type.

**Comment by fppt** [comment]
@aelred can we merge this one? or do you want to look into the generator?

**Comment by aelred** [comment]
@fppt I fixed it on another branch where I'm cleaning the generators.

**Pull request merged**